galley/pkg/config/analysis/local/analyze.go

```diff
@@ -104,7 +110,9 @@ func (sa *SourceAnalyzer) Analyze(cancel chan struct{}) (diag.Messages, error) {
 
 // AddFileKubeSource adds a source based on the specified k8s yaml files to the current SourceAnalyzer
 func (sa *SourceAnalyzer) AddFileKubeSource(files []string, defaultNs string) error {
-	src := inmemory.NewKubeSource(sa.m.KubeSource().Resources())
+	// Note that this source doesn't respect the Disabled flag on KubeResource, but that's OK because
+	// filtering at the source level is about efficiency and the file-based source is super fast
+	src := inmemory.NewKubeSource(sa.kubeResources)
```

Comment:
Is this comment necessarily true? In effect, there is no "disablement" for file based sources.

Reply:
Actually this comment is leftover from a different branch of code and no longer makes sense at all. Removed.